core/server/models/post.js

```diff
@@ -183,7 +192,9 @@ Post = ghostBookshelf.Model.extend({
 
         ghostBookshelf.Model.prototype.saving.call(this, model, attr, options);
 
-        this.set('html', converter.makeHtml(_.toString(this.get('markdown'))));
+        if (this.get('markdown')) {
```

Comment:
What is the usecase for this guard? Atm Ghost doesn't really support not sending markdown?